component/http/cache_metrics.go

```diff
@@ -0,0 +1,162 @@
+package http
```

Comment:
yes, i see your point. The only problem with that is, that if we initialise with init, we will need to pass the 'path' for every metric call ... which i wanted to avoid.

Reply:
also , i am not sure it is the best approach ... It might make the metrics code simpler, but makes the testing more challenging . Will give it a try anyway , and let's see how it looks :)